ui/src/hotkeys.js

```diff
@@ -5,6 +5,7 @@ const keyMap = {
   TOGGLE_PLAY: { name: 'toggle_play', sequence: 'space', group: 'Player' },
   PREV_SONG: { name: 'prev_song', sequence: 'left', group: 'Player' },
   NEXT_SONG: { name: 'next_song', sequence: 'right', group: 'Player' },
+  CURRENT_SONG: { name: 'current_song', sequence: 'c', group: 'Player' },
```

Comment:
You also need to add the translation to the file `ui/src/i18n/en.json` (line ~390)